cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,36 @@
-describe('', () => {
-  before(() => {
-
+describe('Conduit article', () => {
+  let article = {};
+  let user = {};
+  beforeEach(() => {
+    cy.visit('https://conduit.mate.academy/');
+    cy.task('generateUser').then((newUser) => {
+      user = newUser;
+    });
+    cy.task('generateArticle').then((newArticle) => {
+      article = newArticle;
+    });
   });
 
-  it('', () => {
-
+  it('should be able to create a new article', () => {
+    cy.login(user.email, user.username, user.password);
+    cy.reload();
+    cy.get('.navbar').should('contain', user.username.toLowerCase());
+    cy.createArticle(article.title, article.description, article.body);
```

Comment:
In the first test you need to create an article manually. In the second test you can use a function.

Reply:
![image](https://github.com/mate-academy/qa_cypress_e2e_article_flow/assets/140398652/cdf38e23-91ae-44e1-923c-cf56f93f7ddf)
there is mentioned to use a command to login

Reply:
To login, yes. But I'm not talking about `login` function. I am talking about `createArticle` function. You need to create an article manually in the first test. use function only in the second one. 
If you have any questions about that please do not hesitate to ask me on Slack.

Reply:
ok, fixed :)